lib/publiccloud/eks.pm

```diff
@@ -15,6 +15,18 @@
 package publiccloud::eks;
 use Mojo::Base 'publiccloud::aws';
 use testapi;
+use utils;
+use publiccloud::k8s_utils;
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+
+    my $repository //= get_var("PUBLIC_CLOUD_CONTAINER_IMAGES_REPO", 'suse-qec-testing');
+    $self->{repository} = $repository;
+
+    assert_script_run("aws eks update-kubeconfig --name qe-c-testing");
```

Comment:
How long does this take? Is default timeout of 90s enough? Maybe we need to define some timeout to avoid surprises?